cypresse2e/cypress/integration/e2eTests/home/homePage.spec.js

```diff
@@ -1,21 +1,40 @@
+import HomePage from '../../../elements/pages/HomePage';
+
 describe('FightPandemics Home screen', () => {
+
+  var h1HeadingText = "A place to offer and request help";
+  var subHeading1Text = "Pandemics will continue to happen.";
+  var subHeading2Text = "We help communities prepare and respond.";
+  var requestHelpButtonText = 'Request Help'
+  var offerHelpButtonText = 'Offer Help'
+
+  const home = new HomePage();
+
     context('User opens Home screen ', () => {
         beforeEach(() => {
-            cy.visit('');
+          home.visit();
           });
   
-      it('Home screen greetings H1', () => {
-        cy.contains('h1', 'A place to offer and request help')
+      it('Home screen contains headings', () => {
+        home.getH1Heading().contains(h1HeadingText);
+        home.getSubHeading1().contains(subHeading1Text);
+        home.getSubHeading2().contains(subHeading2Text);
       });
 
```

Comment:
Does a test need to be added to check that the "FightPandemics" text right above the h1Heading is visible?

![image](https://user-images.githubusercontent.com/56090896/89360393-4c31ec00-d696-11ea-8c17-f2d410d3b904.png)

Reply:
@jessica-lau 
This text was removed for mobile.
If we want to check it I need to use a selector: 'main > div > div > div > span' to find this element.

Reply:
I think it's okay to leave this out since it doesn't exist on mobile.